api/services/elastigroup/azure/scaleSetVms/responses/startDeployment.yaml

```diff
@@ -0,0 +1,72 @@
+description: Start Deployment Response
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master